pkg/controller/compliancescan/scan.go

```diff
@@ -213,14 +248,210 @@ func newScanPodForNode(scanInstance *compv1alpha1.ComplianceScan, node *corev1.N
 	}
 }
 
+func newPlatformScanPod(scanInstance *compv1alpha1.ComplianceScan, logger logr.Logger) *corev1.Pod {
+
+	mode := int32(0744)
+
+	podName := getPodForNodeName(scanInstance.Name, "api-checks")
+	cmName := getConfigMapForNodeName(scanInstance.Name, "api-checks")
+	podLabels := map[string]string{
+		"complianceScan": scanInstance.Name,
+	}
+
+	return &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: scanInstance.Namespace,
+			Labels:    podLabels,
+			Annotations: map[string]string{
+				"openshift.io/scc": "node-exporter",
+			},
+		},
+		Spec: corev1.PodSpec{
+			ServiceAccountName: resultscollectorSA,
+			InitContainers: []corev1.Container{
+				{
+					Name:  "content-container",
+					Image: getInitContainerImage(&scanInstance.Spec, logger),
+					Command: []string{
+						"sh",
+						"-c",
+						fmt.Sprintf("cp %s /content | /bin/true", path.Join("/", scanInstance.Spec.Content)),
+					},
+					ImagePullPolicy: corev1.PullAlways,
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "content-dir",
+							MountPath: "/content",
+						},
+					},
+				},
+				{
+					Name:  "api-resource-collector",
+					Image: GetComponentImage(API_RESOURCE_COLLECTOR),
+					Args: []string{
+						"--content=/content/" + scanInstance.Spec.Content,
+						"--resultdir=/tmp",
+						"--profile=" + scanInstance.Spec.Profile,
+						"--debug",
+					},
+					ImagePullPolicy: corev1.PullAlways,
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "content-dir",
+							MountPath: "/content",
+						},
+						{
+							Name:      "fetch-results",
+							MountPath: "/tmp",
+						},
+					},
+				},
+			},
+			Containers: []corev1.Container{
+				{
+					Name:  "log-collector",
+					Image: GetComponentImage(LOG_COLLECTOR),
+					Args: []string{
+						"--arf-file=/reports/report-arf.xml",
+						"--results-file=/reports/report.xml",
+						"--exit-code-file=/reports/exit_code",
+						"--oscap-output-file=/reports/cmd_output",
+						"--config-map-name=" + cmName,
+						"--owner=" + scanInstance.Name,
+						"--namespace=" + scanInstance.Namespace,
+						"--resultserveruri=" + getResultServerURI(scanInstance),
+						"--tls-client-cert=/etc/pki/tls/tls.crt",
+						"--tls-client-key=/etc/pki/tls/tls.key",
+						"--tls-ca=/etc/pki/tls/ca.crt",
+					},
+					ImagePullPolicy: corev1.PullAlways,
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "report-dir",
+							MountPath: "/reports",
+							ReadOnly:  true,
+						},
+						{
+							Name:      "tls",
+							MountPath: "/etc/pki/tls",
+							ReadOnly:  true,
+						},
+					},
+				},
+				{
+					Name:    OpenSCAPScanContainerName,
+					Image:   GetComponentImage(OPENSCAP),
+					Command: []string{OpenScapScriptPath},
+					SecurityContext: &corev1.SecurityContext{
+						Privileged: &trueVal,
```

Comment:
Does the pod still need to run privileged if it doesn't mount the host fs?

Reply:
It should not. I've removed the hostroot and scc bits from this part.